src/components/ADempiere/Form/VPOS/Order/index.vue

```diff
@@ -146,13 +146,27 @@
                         <el-col :span="5">
                           <div style="float: right">
                             {{ $t('form.productInfo.price') }}:
-                            <b>{{ formatPrice(scope.row.product.priceStandard, pointOfSalesCurrency.iSOCode) }}</b>
+                            <b>
```

Comment:
Just can see change for source code with the same functionality, please let it without changes